db/fixtures/miq_user_roles.yml

```diff
@@ -352,6 +358,8 @@
   - ems_physical_infra_show_list
   - ems_physical_infra_tag
   - ems_physical_infra_timeline
+  - event_stream_show_list
```

Comment:
Not really, I just wondered if the user permissions are accurate but I just noticed @h-kataria is on it.